### PR TITLE
SRVCOM-1272 Report UI tests in junit format

### PR DESCRIPTION
**Pull request by mgencur**

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -50,7 +50,7 @@ test-e2e:
 	./test/e2e-tests.sh
 
 # Run E2E tests from the current repo for serving+eventing+knativeKafka
-test-e2e-with-kafka:
+test-e2e-with-kafka: test-ui-e2e
```
Comment (cardil) [inline Makefile:53]: This is temporal change, right?

/hold

Comment (mgencur) [reply Makefile:53]: Right. No need to hold. This PR is WIP

**Comment by mgencur** [comment]
The previous run correctly reports the failed tests as junit: https://prow.ci.openshift.org/view/gs/origin-ci-test/pr-logs/pull/openshift-knative_serverless-operator/874/pull-ci-openshift-knative-serverless-operator-main-4.7-operator-e2e-aws-ocp-47/1379427360650039296
I've just squashed the changes and removed the temporary commit. The UI tests won't run in pre-submits now.

**Comment by cardil** [comment]
The primary goal of reporting all UI tests as xUnit is achieved, so I can give lgtm.

I don't like the results are also printed on console as bloody XML. That's ugly. I rather, like to see a nice human-readable output like default `spec` reporter. That would involve using multiple reporters like in this example: https://docs.cypress.io/guides/tooling/reporters#Spec-to-STDOUT-save-JUnit-XML-files. We can do it in follow-up PR, if you like...

**Comment by mgencur** [comment]
Removed the temporary commit. The PR is ready for another review.

**Review by cardil — APPROVED** [review]
Looks great. Thanks.

/lgtm

**Pull request merged**